src/billing/views.py

```diff
@@ -57,8 +57,8 @@ def get(self, request):
 
         messages.add_message(
             self.request, messages.SUCCESS,
-            _("The billing with the identifier #%s \
-            has been successfully created." % billing.id)
+            _("The billing with the identifier #%s "
+              "has been successfully created." % billing.id)
```

Comment:
@lingxiaoyang What do you thin about increase the limit of the code line... we use pep8 and the limit is 79, my appreciation it's so small, we need to spit a lot of code lines. Django use more than 79, exactly [119](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).